### PR TITLE
Flake it!

### LINKED ISSUE #221: Flake it
The #215 PR has been opened to integrate a flake8 check on the code and to travis! Please feel free to address any issue and help us implement some code improvements!

### PR DESCRIPTION
**Pull request by natct10**
Brace yourself, errors are coming! But please let me know if you want to ignore some of the warnings.
Fix  #221
- [x] Cleaup imports (order & remove unused)
- [x] remove whitespaces
- [x] try to break lines longer than 80 characters when possible

### CODE REVIEW

**Comment by bowni** [comment]
OK, good job adding it in the CI, it will help enforce proper coding standards. I'd say we keep this PR open until the CI passes. Who wants to give a try at the bad formatting issues that Flake points out?

**Comment by RomainGoussault** [comment]
Is there a way to automatically fix the easy ones (whitespaces, lines too long, etc ?)

**Comment by RomainGoussault** [comment]
Only 20 warning left

**Review by RomainGoussault — APPROVED** [review]
Well done !

**Comment by natct10** [comment]
> Well done !

OH, awesome, thx @RomainGoussault !!
+1 for the auto-congrats :D

**Review by natct10 — COMMENTED** [review]
I can't approve as I've been involved in the PR but you have my full endorsement and respect for what you did here!

**Review by bowni — APPROVED** [review]
OK, wow... impressive @RomainGoussault and @natct10 !!! Thank you for having had the courage... :sunglasses: 
What are the two inline comments starting with `# noqa E...`?

**Comment by natct10** [comment]
> OK, wow... impressive @RomainGoussault and @natct10 !!! Thank you for having had the courage... sunglasses
> What are the two inline comments starting with `# noqa E...`?

`noqa` is a flake8 ignore keyword, [see](https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html), that you can disable with flake8 --disable-noqa  :troll:

File: datasets/dataset_titanic.py [hunk 1]
```diff
@@ -5,37 +5,36 @@
 """
 
 from sklearn.model_selection import train_test_split
-from keras.models import Sequential
-from keras.layers import Dense, Dropout
```
Comment (natct10) [inline datasets/dataset_titanic.py:9]: thx!

File: datasets/dataset_titanic.py [hunk 2]
```diff
@@ -5,37 +5,36 @@
...
 import pandas as pd
 import numpy as np
-import keras
```
Comment (natct10) [inline datasets/dataset_titanic.py:12]: +1

**Review by arthurPignet — APPROVED** [review]

**Pull request merged**